guides/source/developers/customizations/customizing-attributes.html.md

```diff
@@ -0,0 +1,92 @@
+# Customizing Model Attributes
+
+This guide covers how to add editable attributes to models in Solidus.
+We will cover an example of adding a new attribute to the `Spree:Taxon` model 
+to allow for an additional text area.
+
+-   Adding attribute columns to the database
+-   Adding attribute fields to the admin panel
+-   Allowing custom attributes to be saved
+
+## Adding Columns to the Database
+
+Columns can be added to Solidus database tables using standard Rails migrations
+[standard Rails migrations][standard_rails_migrations].
+
+If you wanted to add a `more_information` attribute, you could run the 
+following command:
+
+```bash
+rails generate migration AddMoreInformationToTaxons
+``` 
+
+This would create a migration file starting with a timestamp 
+`db/migrate/20190925192026_add_more_information_to_taxons.rb`. Then update the 
+file with the following contents to add a new text type column 
+`more_information` to the table:
+
+```ruby
+class AddMoreInformationToTaxons < ActiveRecord::Migration[5.2]
+    def change
+        add_column :spree_taxons, :more_information, :text
+    end
+end
+```
+
+After you are done editing your migration run the following command to add the 
+column to the database:
+
+```bash
+rails db:migrate
+```
+
+[standard_rails_migrations]: https://edgeguides.rubyonrails.org/active_record_migrations.html
+
+## Adding Fields to the Admin Panel
+
+We have a complete guide on [customizing the admin panel][admin], but will
+go over it here briefly for our example. While that guide covers template
+replacement and using the [Deface][deface] gem, we'll just cover the 
+Deface approach. This allows changing parts of the views without overriding
+ them entirely.
+
+To add a new form field for a custom attribute to the taxon form, we would
+want to extend `app/views/spree/admin/taxons/_form.html.erb` from the 
+Solidus backend. We can do that with Deface by creating a file 
+`app/overrides/spree/admin/taxons/_form/add_fields.html.erb.deface`.
+
+Below is an example of adding a new field to this form for a `more_information`
+field.
+
+```erb
+<!-- insert_bottom '[data-hook=admin_inside_taxon_form]' -->
+<div class="col-10">
+    <%= f.field_container :more_information do %>
+        <%= f.label :more_information %><br />
+        <%= f.text_area :more_information, class: 'fullwidth', rows: 6 %>
+    <% end %>
+</div>
+```
+
+[admin]: customizing-admin.html
+[deface]: https://github.com/spree/deface
+
+## Allowing Custom Attributes to be Saved
+
+You'll notice after adding a new database column and form field for a custom
+attribute, that it won't hold a value after submitting the form. This is due
```

Comment:
Please take it with a grain of salt, I'm not a native English speaker:
I think this sentence is a bit difficult to understand. Maybe just repositioning some parts will make it clearer. What do you think?

Reply:
This whole paragraph has been updated, let me know if it reads better.